js/components/wysiwyg.js

```diff
@@ -145,49 +79,43 @@ Fliplet.FormBuilder.field('wysiwyg', {
 
     var config = {
       target: this.$refs.textarea,
-      theme: 'modern',
```

Comment:
@tonytlwu @simonchapman1986 I removed the theme property to prevent potential issues with different versions of TinyMCE.